doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-rules-WHEN-proc.adoc

```diff
@@ -44,11 +44,11 @@ Bankruptcy(application == a ).
 +
```

Comment:
I see the snippet `Bankruptcy(application == a ).` was not touched, however would you mind to update to `Bankruptcy( application == a ).` ?